signerverifier/utils_test.go

```diff
@@ -0,0 +1,39 @@
+package signerverifier
+
+import (
+	"path/filepath"
+	"testing"
+	"os"
+	"github.com/stretchr/testify/assert"
+)
+func TestLoadKeyFromSSLibBytes(t *testing.T) {
+	t.Run("RSA public key", func(t *testing.T) {
+		contents, err := os.ReadFile(filepath.Join("test-data", "rsa-test-key.pub"))
+		if err != nil {
+			t.Fatalf("bad test")
```

Comment:
```suggestion
			t.Fatal(err)
```
https://github.com/secure-systems-lab/go-securesystemslib/blob/8c0fa8d1979fda745dae5325708417965522313a/signerverifier/ecdsa_test.go#L164

Reply:
fixed